main.go

```diff
@@ -85,6 +85,7 @@ func main() {
 	flag.StringVar(&settings.HTTPUserAgent, "httpuseragent", "", "sets the HTTP user agent")
 	flag.StringVar(&settings.HTTPProxy, "httpproxy", "", "sets the HTTP proxy server")
 	flag.BoolVar(&settings.EnableExchangeHTTPDebugging, "exchangehttpdebugging", false, "sets the exchanges HTTP debugging")
+	flag.DurationVar(&settings.TradeBufferProcessingInterval, "tradeprocessinginterval", time.Second*15, "sets the interval to save trade buffer data to the database")
```

Comment:
Can use trade.DefaultProcessorIntervalTime, similar to how we do it with request.DefaultMaxRequestJobs